Gemfile

```diff
@@ -20,22 +20,19 @@ gem 'uglifier'
 gem 'newrelic_rpm'
 gem 'rollbar'
 
-# Api
-gem 'redis-rails'
-
 # Utilities
+gem 'redis-rails'
 gem 'rest-client'
 
 group :development, :test do
   gem 'dotenv-rails'
   gem 'pry-byebug'
+  gem 'pry-rails'
   gem 'rack-mini-profiler'
   gem 'rspec-rails'
 end
 
 group :development do
-  gem 'listen'
```

Comment:
konzola sa da spustit? nedavno som narazil na to ze ked som vyrazil `listen` tak to popadalo

Reply:
@pavolzbell  ak myslis `rails console` a console v browseri na error pages tak funguje

Reply:
tak to bude asi ok